vignettes/including-data-in-teal-applications.Rmd

```diff
@@ -0,0 +1,251 @@
+---
+title: "Including Data in teal Applications"
+author: "NEST CoreDev"
+output: 
+  bookdown::html_document2:
```

Comment:
why bookdown needed?

Reply:
> * added `bookdown` to `Suggests` to allow for cross-references within document

Reply:
Can't you do it with pure markdown syntax https://linuxhint.com/markdown-link-section/?

```
(jump to [`teal_data` properties](#teal-data-properties))
```

```
(back to [Reproducible data](#reproducible-data))
```

Reply:
Apparently I can.